data/solid-oidc.jsonld

```diff
@@ -97,7 +97,7 @@
     "comment": "The server metadata MUST indicate support for the webid claim in the claims_supported field.",
     "requirementSubject": "OpenIDProvider",
     "requirementLevel": "MUST",
-    "requirementReference": "https://solid.github.io/solid-oidc/#discovery" },
+    "requirementReference": "https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata" },
```

Comment:
We should be linking to Solid-OIDC here

Reply:
But https://solid.github.io/solid-oidc/#discovery says nothing about `claims_supported`

Reply:
So maybe that is the root of the problem :)